### ISSUE TITLE
Is trunkmaster OK?

### ISSUE THREAD

**Issue by felixding** [trunkmaster/nextspace#413]
He is in Kyiv and the last commit is four days ago. Is everything alright?

**Comment by roblillack** [comment]
There's already a discussion from yesterday which is unanswered as of yet: https://github.com/trunkmaster/nextspace/discussions/412

Let's hope that he and his family are okay.

**Comment by uchuugaka** [comment]
Pretty sure Ukraine citizens are busy these days with Putin‘s troublemaking. I hope everyone there can avoid any further suffering.

**Comment by zeroheure** [comment]
Yes he is ok ! he just answered https://github.com/trunkmaster/nextspace/discussions/412 and on the Gnustep mailing list!

**Comment by trunkmaster** [comment]
@felixding thank you! I'm fine.
